src/selectors/scope.test.js

```diff
@@ -86,9 +91,10 @@ describe("currentScopeSelector", () => {
 			[state],
 			"to equal",
 			Immutable.fromJS({
-				name: "Global",
-				id: "Global",
-				children: ["FirstChild", "SecondChild"],
+				name: "Premier fils",
```

Comment:
Address this in the test name, and make sure all three outcomes of the selector are tested.